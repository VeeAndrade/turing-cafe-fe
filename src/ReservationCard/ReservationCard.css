.reservation-card {
  border: solid 4px #008080;
  border-radius: 15px;
  width: 250px;
  height: 200px;
  margin-bottom: 40px;
  background-color:#fff;
}

.reservation-detail {
  color: #008080;
}

.cancel-reservation-button {
  width: 80%;
  height: 30px;
  background-color: #008080;
  color: #fff;
}