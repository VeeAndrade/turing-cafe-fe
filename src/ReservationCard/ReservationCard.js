import React from 'react';
import './ReservationCard.css'

const ReservationCard = ({ reservation }) => {
return (
  <section key={reservation.id} className='reservation-card'>
    <h3 className='reservation-detail'>{reservation.name} </h3>
    <p className='reservation-detail'>{reservation.date}</p>
    <p className='reservation-detail'>{reservation.time} pm</p>
    <p className='reservation-detail'>Number of guest: {reservation.number} </p>
    <button className='cancel-reservation-button'>Cancel</button>
  </section>
  )
}

export default ReservationCard;