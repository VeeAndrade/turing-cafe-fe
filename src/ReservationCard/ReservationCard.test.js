import React from 'react';
import { shallow } from 'enzyme';
import ReservationCard from './ReservationCard';

describe('ReservationCard', () => {

  it('should match the snapshot with all the data passed in correctly', () => {
    const mockReservation = {
      id: 42,
      name: 'Vee',
      date: '06/21',
      time: '7:00',
      number: 3
    }
    const wrapper = shallow(<ReservationCard reservation={mockReservation} />)

    expect(wrapper).toMatchSnapshot();
  });
});