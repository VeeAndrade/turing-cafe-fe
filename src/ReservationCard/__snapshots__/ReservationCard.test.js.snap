// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReservationCard should match the snapshot with all the data passed in correctly 1`] = `
<section
  className="reservation-card"
  key="42"
>
  <h3
    className="reservation-detail"
  >
    Vee
     
  </h3>
  <p
    className="reservation-detail"
  >
    06/21
  </p>
  <p
    className="reservation-detail"
  >
    7:00
     pm
  </p>
  <p
    className="reservation-detail"
  >
    Number of guest: 
    3
     
  </p>
  <button
    className="cancel-reservation-button"
  >
    Cancel
  </button>
</section>
`;
